force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -271,11 +331,19 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer findWordInList(String wordToFind, List<String> words) {
         // The variable to store the index of the word
-
-        // Loop through the list of words
-            // If the current word is the word to find, exit the loop
-
-        return null; // Replace null with the variable you used to store the result
+        Integer position = 0; 
+        for (String word : words) {
+            if(word == wordToFind) {
+                position = words.indexOf(word);
```

Comment:
If you are going to use this indexOf method, you really dont need to loop at all, it returns what you want in this question. Otherwise you would need to increment a counter to know which loop/position you are in.